new/NAR0205StanzaCaliph.xml

```diff
@@ -0,0 +1,60 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="NAR0205StanzaCaliph" xml:lang="en" type="nar">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>A stanza corresponding to the mircale of the Caliph of Atrib</title>
```

Comment:
This is rather a description and should be encoded as abstract. For the title, I suggest "Miracle of the Caliph of Atrib". What do you mean with "corresponding"? Why not "Stanza about the miracle of the Caliph of Atrib"?